### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by Cadair**
mostly from @mbobra

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,19 +1,27 @@
 The SunPy project is happy to announce the release of SunPy 0.8.0.
 
-SunPy 0.8 is the next major release of SunPy and contains 1390 commits in 200
-pull requests, closing 163 issues from 34 people, 16 of which have never
-contributed before. 
+SunPy 0.8 is the next major release of SunPy and contains 1442 commits in over
+200 pull requests, closing 163 issues from 35 people, 17 of which have never
+contributed before.
```
Comment (nabobalis) [inline RELEASE.md]: These include this?!

Comment (Cadair) [inline RELEASE.md]: the commit number does yes.

File: docs/guide/acquiring_data/fido.rst [hunk 2]
```diff
@@ -47,16 +48,20 @@ variable set to the Fido search, in this case, result::
     2012-03-04 00:00:00 2012-03-06 00:00:00 Proba2       lyra        nan
 
 Queries can be made more flexible or specific by adding more attrs objects to
-the Fido search. Specific passbands taken with given instruments can be searched
-for by supplying an `~astropy.units.Quantity` to the Wavelength attribute::
+the Fido search. To search for specific passbands can be searched for by
```
Comment (nabobalis) [inline docs/guide/acquiring_data/fido.rst]: To search ... can be searched?

File: docs/guide/acquiring_data/fido.rst [hunk 3]
```diff
@@ -47,16 +48,20 @@ variable set to the Fido search, in this case, result::
...
 
     >>> import astropy.units as u
     >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('norh'),
                              a.Wavelength(17*u.GHz))
 
-Data of a given cadence can also be specified using the Sample attribute. As
-`~sunpy.net.attrs.vso.Sample` is only supported by the `sunpy.net.vso.VSOClient`
-you specify sample with ``a.vso.Sample``. Attributes like this which are client
+Data of a given cadence can also be specified using the Sample attribute.
+
```
Comment (nabobalis) [inline docs/guide/acquiring_data/fido.rst]: new line?

File: docs/index.rst [hunk 4]
```diff
@@ -6,7 +6,6 @@ free and open-source solar data analysis environment for Python.
 
 .. toctree::
   :maxdepth: 2
-  :titlesonly:
```
Comment (nabobalis) [inline docs/index.rst]: What stupid person added that. #REVIEWBETTERPEPEOPLE

File: RELEASE.md [hunk 5]
```diff
@@ -1,19 +1,27 @@
 The SunPy project is happy to announce the release of SunPy 0.8.0.
 
-SunPy 0.8 is the next major release of SunPy and contains 1390 commits in 200
-pull requests, closing 163 issues from 34 people, 16 of which have never
-contributed before. 
+SunPy 0.8 is the next major release of SunPy and contains 1442 commits in over
+200 pull requests, closing 163 issues from 35 people, 17 of which have never
```
Comment (SolarDrew) [inline RELEASE.md]: of whom

File: docs/guide/acquiring_data/fido.rst [hunk 6]
```diff
@@ -21,8 +21,9 @@ Searching for Data Using Fido
 -----------------------------
 
 To search for data with Fido, you need to specify attributes to search against.
-Examples of these attributes are Time, Instrument, Wavelength. Enter these
-properties using SunPy's attrs module::
+Examples of these attributes are Time, Instrument, Wavelength, some of these
+attributes are client specific, such as `wibble <sunpy.net.vso.attrs>` or
```
Comment (SolarDrew) [inline docs/guide/acquiring_data/fido.rst]: wibble?

File: docs/guide/acquiring_data/fido.rst (hunk 2 above)
Comment (SolarDrew) [inline docs/guide/acquiring_data/fido.rst]: "the Fido search. Specific passbands can be searched for by..."

File: docs/guide/acquiring_data/fido.rst (hunk 6 above)
Comment (Cadair) [inline docs/guide/acquiring_data/fido.rst]: lmao can you tell I did this at 1am

Comment (SolarDrew) [inline docs/guide/acquiring_data/fido.rst]: All the best code is written at 1am.

Comment (nabobalis) [inline docs/guide/acquiring_data/fido.rst]: OH that wasnt real

File: docs/guide/data_types/maps.rst [hunk 7]
```diff
@@ -40,11 +38,12 @@ may vary. SunPy can also create maps from the jpg2000 files from
 Creating Custom Maps
 --------------------
 It is also possible to create maps using custom data (e.g. from a simulation).
-To do this you need to provide `~sunpy.map.map_factory.MapFactory` with both the data array as
-well as some basic meta information. If no header is given then some default
-values as assumed. Here is a simple example::
+To do this you need to provide `sunpy.map.Map <sunpy.map.map_factory.MapFactory>`
+with both the data array as well as some basic meta information. If no header is
+given then some default values as assumed. Here is a simple example::
```
Comment (SolarDrew) [inline docs/guide/data_types/maps.rst]: values are assumed

File: docs/guide/tour.rst [hunk 8]
```diff
@@ -60,9 +61,12 @@ create some fake data and pass it into a timeseries object.
     my_timeseries = ts.TimeSeries(sunpy.data.sample.GOES_XRS_TIMESERIES, source='XRS')
     my_timeseries.peek()
 
-We've created this timeseries object by passing TimeSeries a string which represent the name of a GOES lightcurve file.
-The ``.peek()`` method plots the timeseries data and displays the plot with some default settings.
-You can also use ``my_timeseries.plot()`` if you want more control over the style of the output plot.
+We've created this timeseries object by passing TimeSeries a string which
+represent the name of a GOES lightcurve file. The
```
Comment (SolarDrew) [inline docs/guide/tour.rst]: represents

**Pull request merged**